spec/test-data/translations/strings-with-escape-character-in-root-context.strings

```diff
@@ -0,0 +1,10 @@
+/* This file has characters root context (not in a comment, or key, or value), which is not a valid `.strings` syntax */
+"key" = "value";
+"other key" = "other value";
+
+/* Escaped characters should be allowed in block comments \\ */
+"yet another key" = "yet another value"; // Escapes should be allowed in inline comments, too \\
```

Comment:
I was about to comment in the PR that we should add a case of a `\\` in an inline comment as well… but then just saw you thought of that too with this case here 👍 👌